#request
uint8 payload_index
uint8 exposure_mode
uint8 shutter_speed
---
#response
bool result